OutSystems/outsystems-datagrid
# ROU-4320: Fix setRowAsSelected not working in a Grid with RowCheckboxes

This PR is for add the functionality to select rows of a Grid with RowCheckboxes via API

### What was happening
* The SetRowAsSelected Client Action did not work in a Grid with RowCheckboxes.

### What was done
* In setRowAsSelected method of Selection feature:
    * Removed the error message when using SetRowAsSelected in a Grid with RowCheckboxes.
    * Set the isChecked row metadata and the isSelected row flag to the desired state.
    * Refresh the CollectionView to trigger the updatingView event and change the checkboxes state

### Test Steps
1. Go to the test page with a Grid with RowCheckboxes
2. Select the Rows via SetRowAsSelected Client Action
3. Check if the desired rows are checked


### Checklist
